definitions/variable/technology/technologies.yaml

```diff
@@ -492,3 +496,60 @@
     description: Levelized cost of storage for {Storage Type}
     unit: [MEUR_2020/MWh, MUSD_2010/MWh, MEUR_2020/PJ, MUSD_2010/PJ]
     skip-region-aggregation: true
+- Capital Cost|Electricity|Diesel|Stand-alone:
+    description: The capital cost (per Capacity unit) of a stand- alone diesel generator.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Electricity|Solar PV|Stand-alone:
+    description: The capital cost (per Capacity unit) of a stand- alone PV module.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Electricity|Diesel|Mini Grid:
+    description: The capital cost (per Capacity unit) of a mini Grid diesel generator.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Electricity|Solar PV|Mini Grid:
+    description: The capital cost (per Capacity unit) of a mini Grid PV system.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Electricity|Wind|Mini Grid:
+    description: The capital cost (per Capacity unit) of a mini Grid wind powered system.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Electricity|Hydro|Mini Grid:
+    description: The capital cost (per Capacity unit) of a mini Grid hydropower system.
+    unit: US$2020/kW
+    skip-region-aggregation: true
+- Capital Cost|Batteries:
+    description: Expected hybrid mini-grid component costs batteries.
+    unit: US$2020/kWp
```

Comment:
You have to set `skip-region-aggregation: true` here. Reason being that the default region aggregation simply adds up all the values for across different regions. For a variable with a unit like `US$2020/kWp` this would lead to wrong values. 
We do offer the possibility of using a weighted average but for now, for simplicity let's skip region aggregation.
The same logic applies for all variables with `US$2020/kWp` and `US$2020/km` as a unit.